gnocchi/indexer/sqlalchemy_base.py

```diff
@@ -198,7 +198,7 @@ def to_baseclass(self):
 
 
 class ResourceJsonifier(indexer.Resource):
-    def jsonify(self):
+    def jsonify(self, json_attrs=None):
```

Comment:
The name of the method is `jsonify` so calling the argument `attrs` should be enough. But not a big deal. :)